pinot-core/src/test/java/org/apache/pinot/core/data/manager/BaseTableDataManagerAcquireSegmentTest.java

```diff
@@ -240,7 +239,7 @@ public void testReplace()
 
     runStorageServer(numQueryThreads, runTimeSec, tableDataManager);  // replaces segments while online
 
-//    System.out.println("Nops = " + _numQueries + ",nDrops=" + _nDestroys + ",nCreates=" + _allSegments.size());
+    // System.out.println("Nops = " + _numQueries + ",nDrops=" + _nDestroys + ",nCreates=" + _allSegments.size());
```

Comment:
Remove it

Reply:
Instead, print out the random seed  once when the test starts. That way, if the test fails, we will be able to reproduce the failure by manually setting the seed to the value printed.

Reply:
@mcvsubbu thanks for reviewing. about this suggestion, I didn't find a good way to get seed. Instead, it's simpler to set a seed and print it out for record. btw, I see a local random object is used in testReplace() method, but a constant RANDOM object is used in other methods. Is this intended?

Reply:
Probably un-intended. Random is random, no matter if there is one or many. I suggest you set the seed to be the current time, and print out that value once in the test. Refactor the test to use the same random everywhere